certification/internal/shell/has_license.go

```diff
@@ -8,40 +8,52 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	licensePath         = "/licenses"
+	newLine             = "\n"
+	minLicenseFileCount = 1
+)
+
 type HasLicenseCheck struct{}
 
 func (p *HasLicenseCheck) Validate(image string) (bool, error) {
-	line, err := p.getDataToValidate(image)
+	licenseFileList, err := p.getDataToValidate(image)
 	if err != nil {
 		return false, err
 	}
-	return p.validate(line)
+	return p.validate(licenseFileList)
 }
 
 func (p *HasLicenseCheck) getDataToValidate(image string) (string, error) {
 	runOpts := cli.ImageRunOptions{
-		EntryPoint:     "ls",
-		EntryPointArgs: []string{"-A", "/licenses"},
+		EntryPoint:     "stat",
+		EntryPointArgs: []string{licensePath},
 		LogLevel:       "debug",
 		Image:          image,
 	}
 	runReport, err := podmanEngine.Run(runOpts)
 	if err != nil {
-		ok, _ := p.validate(runReport.Stdout)
-		if ok {
-			log.Error("some error attempting to identify if /licenses container the license: ", err)
-			log.Debugf("Stdout: %s", runReport.Stdout)
-			log.Debugf("Stderr: %s", runReport.Stderr)
-			return "", err
-		}
-
+		log.Error("Error when checking for "+licensePath+" : ", err)
```

Comment:
I think using fmt.Sprintf for the first arg improves the readability.

Reply:
Agreed

Reply:
Agree as well